### PR TITLE
Unify Tree Fluid Outputs

### PR DESCRIPTION
**Pull request by MuteTiefling**
Corrects #2621 - see `cutting.js`

Unify fluid outputs of logs between IF and Thermal.
Increases rates for IF compared to default. Acacia produces double the latex now. Stripped logs are worth half their barky counterparts instead of 1mb per operation.
Rebalances some fluids to make more sense (more resin from evergreen type trees, less from others)

I'm not super happy with the search function to find the stripped log type, but I couldn't come up with any other way to do it. It's not _too_ heavy on run time, however. The entire IF script using it takes .001 seconds to run.

### CODE REVIEW

File: kubejs/server_scripts/enigmatica/kubejs/functions.js [hunk 1]
```diff
@@ -46,6 +46,16 @@ function wrapArray(array) {
...
 
+function getStrippedLog(value) {
+    let strippedLog = air;
+    for (var i = 0; i < buildWoodVariants.length; i++) {
+        if (buildWoodVariants[i].logBlock === value) {
+            strippedLog = buildWoodVariants[i].logBlockStripped;
+        }
+    }
+    return strippedLog;
+}
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: May I suggest a different approach?

```js
function getStrippedLogFrom(logBlock) {
    buildWoodVariants.find((wood) => {
        if (wood.logBlock == logBlock) {
            return wood.logBlockStripped;
        }
    });
}

Comment (Buuz135) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: I think there is a return missing in `getStrippedLogFrom` with your method @NillerMedDild

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: ah true, the air part is needed

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: yeah some trees don't have a stripped block, like fruit trees. so I need air as a default. It's more the actual search that I don't like. Seems like this being an array of objects I shouldn't have to iterate on it...

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: which, of course, you're showing
lol

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: Find still iterates it I'm afraid :/

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: Yeah, not getting any matches with this, unfortunately. I'm ending up with Air as a result for every request.

```js
function getStrippedLogFrom(logBlock) {
    buildWoodVariants.find((wood) => {
        console.log(`Looking for: ${logBlock}`);
        if (wood.logBlock == logBlock) {
            console.log(`Found: ${wood.logBlockStripped}`);
            return wood.logBlockStripped;
        } else {
            console.log(`Not Found: ${logBlock}`);
            return air;
        }
    });
}
```
snippet of the log:
```
[15:49:31] [Server thread/INFO]: server_scripts:enigmatica/kubejs/functions.js:61: Looking for: byg:cika_log
[15:49:31] [Server thread/INFO]: server_scripts:enigmatica/kubejs/functions.js:66: Not Found: byg:cika_log
```

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: Oh alright, sorry about that 😅

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: Actually.. You're only checking the first variant.

Comment (NillerMedDild) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]:
```js
function getStrippedLogFrom(logBlock) {
    buildWoodVariants.find((wood) => {
        console.log(`Looking for: ${logBlock}`);
        if (wood.logBlock == logBlock) {
            console.log(`Found: ${wood.logBlockStripped}`);
            return wood.logBlockStripped;
        }
    });
    console.log(`Not Found: ${logBlock}`);
    return air;
}

Comment (MuteTiefling) [reply kubejs/server_scripts/enigmatica/kubejs/functions.js:57]: Oooooh.

File: kubejs/server_scripts/enigmatica/kubejs/functions.js [hunk 2]
```diff
@@ -46,14 +46,14 @@ function wrapArray(array) {
...
-function getStrippedLog(value) {
-    let strippedLog = air;
-    for (var i = 0; i < buildWoodVariants.length; i++) {
-        if (buildWoodVariants[i].logBlock === value) {
-            strippedLog = buildWoodVariants[i].logBlockStripped;
+function getStrippedLogFrom(logBlock) {
+    let result = air;
+    buildWoodVariants.find((wood) => {
+        if (wood.logBlock == logBlock) {
+            result = wood.logBlockStripped;
```
Comment (NillerMedDild) [inline kubejs/server_scripts/enigmatica/kubejs/functions.js:53]: The find function will continue searching after it has found it's result in this form, there needs to be a `break;` or `return result` here :)

**Comment by MuteTiefling** [comment]
oop, just found another issue, hold on

**Review by NillerMedDild — APPROVED** [review]

**Pull request merged**